controllers/dspipeline_params.go

```diff
@@ -243,6 +248,19 @@ func (p *DSPAParams) RetrieveOrCreateObjectStoreSecret(ctx context.Context, clie
 	return accessKey, secretKey, nil
 }
 
+func getDBExtraParams(tls string, log logr.Logger) (string, error) {
```

Comment:
DBExtraparams can contain a lot more than just `tls` parameter, so I suggest doing the following: 

1. either keep this specific to just the default setting, and rename this to `getDefaultDBExtraParams`, and put this in the `defaults.go` folder
2. or make this more generic, and maybe use a `map[string]string`, could maybe use it [here](https://github.com/opendatahub-io/data-science-pipelines-operator/pull/620/files#diff-5fd0ead87d572dec71ec0194647f49e295c220aa3798b993d75e3b491f64a17aL324-L330) too